crates/cli/src/lib.rs

```diff
@@ -271,9 +282,9 @@ pub fn build_app<'a>() -> Command<'a> {
             Command::new(CMD_DOCS)
                 .about("Generate documentation for a Roc package")
                 .arg(Arg::new(ROC_FILE)
-                    .multiple_values(true)
                     .help("The package's main .roc file")
-                    .allow_invalid_utf8(true)
+                    .num_args(0..0)
```

Comment:
0 to 0 ?

Reply:
I... do not know how that got there. Thanks for catching it.